src/components/ComboButton/_mixins.scss

```diff
@@ -0,0 +1,95 @@
+////
+/// Combo button component.
+/// @group combo-button
+/// @copyright IBM Security 2019
+////
+
+@mixin combo-button {
+  &__wrapper {
+    position: relative;
+    display: flex;
+    justify-content: flex-end;
+
+    & .#{$prefix}--btn--primary:first-child {
```

Comment:
For styling purposes of this component, I would suggest that combo-button classes are better than repurposing component classnames that we don't have control over.